orchagent/orch.cpp

```diff
@@ -214,14 +229,44 @@ void Orch::doTask()
     }
 }
 
-void Orch::dumpTuple(Consumer &consumer, KeyOpFieldsValuesTuple &tuple)
+// TODO: Move to swss-common library
+string getTimestamp()
+{
+    char buffer[64];
+    struct timeval tv;
+
+    gettimeofday(&tv, NULL);
+
+    size_t size = strftime(buffer, 32 ,"%Y-%m-%d.%T.", localtime(&tv.tv_sec));
+
+    snprintf(&buffer[size], 32, "%06ld", tv.tv_usec);
+
+    return string(buffer);
+}
+
+void Orch::recordTuple(Consumer &consumer, KeyOpFieldsValuesTuple &tuple)
```

Comment:
is the format same as saiplayer format? can we reuse the saiplayer tool to replace the appdb record?